services/visual-studio-marketplace/visual-studio-marketplace-release-date.tester.js

```diff
@@ -0,0 +1,79 @@
+'use strict'
+
+const t = (module.exports = require('../tester').createServiceTester())
+const { isFormattedDate } = require('../test-validators')
+
+t.create('date')
+  .get('/visual-studio-marketplace/release-date/yasht.terminal-all-in-one.json')
+  .expectBadge({
+    label: 'release date',
+    message: isFormattedDate,
+  })
+
+t.create('date')
+  .get('/visual-studio-marketplace/release-date/yasht.terminal-all-in-one.json')
+  .intercept(nock =>
+    nock('https://marketplace.visualstudio.com/_apis/public/gallery/')
+      .post(`/extensionquery/`)
+      .reply(200, {
+        results: [
+          {
+            extensions: [
+              {
+                statistics: [
+                  {
+                    statisticName: 'averagerating',
+                    value: 2.5,
+                  },
+                  {
+                    statisticName: 'ratingcount',
+                    value: 10,
+                  },
+                ],
+                versions: [
+                  {
+                    version: '1.0.0',
+                  },
+                ],
+                releaseDate: '2013-04-13T07:50:27.000Z',
+                lastUpdated: '2019-04-13T07:50:27.000Z',
+              },
+            ],
+          },
+        ],
+      })
+  )
+  .expectBadge({
+    label: 'release date',
+    message: 'april 2013',
+    color: 'red',
+  })
```

Comment:
Same comment as the similar test in the last updated service tester file, let's remove this

```suggestion
```

Reply:
Removed